gallery/models.py

```diff
@@ -25,7 +25,7 @@ class Meta:
 
 
 class Album(models.Model) :
-    title = models.CharField(max_length=200)
+    title = models.CharField(max_length=200, unique=True)
```

Comment:
Why do you need to make it unique? there is already an ID which will act as the primary key.